tests/test_launcher.py

```diff
@@ -34,6 +34,32 @@ def test_get_sherlock_exe_path(self, mock_get_base_ansys):
             launcher._get_sherlock_exe_path(),
         )
 
+    def test_extract_sherlock_version_year_with_two_digits(self):
+        with self.assertRaises(ValueError) as context:
+            launcher._extract_sherlock_version_year(24)
```

Comment:
I'd use 999 for the year value. That ensures that the boundary condition is met.